### ISSUE TITLE
Make datepicker accessible

### ISSUE THREAD

**Issue by nosvalds** [icehaunter/vue3-datepicker#91]
Using the https://wave.webaim.org/ tool, I get a number of "Empty button" errors with the datepicker. There are probably other improvements that can be made as well with aria attributes and the like.

![image](https://user-images.githubusercontent.com/60047271/167613840-7c52a061-b00c-4cc9-9ed2-945710077f24.png)

It would be awesome if accessibility features were built into this datepicker. Here are some good examples of how to do this: https://www.webaxe.org/accessible-date-pickers/

**Comment by Gyurmatag** [comment]
Completely agree with this feature request! When this will happen?

**Comment by icehaunter** [comment]
Yeah, I agree that would be good, however I'm not an active user of the datepicker anymore, so I am very unlikely to implement it myself. Contributions welcome!
